{"status":"OK","result":{"originalLocale":"en","allowViewHistory":true,"creationTimeSeconds":1523468887,"rating":-6,"authorHandle":"shivamgor498","modificationTimeSeconds":1523468887,"id":58877,"title":"\u003cp\u003eRecieving Runtime Error\u003c/p\u003e","locale":"en","content":"\u003cdiv class\u003d\"ttypography\"\u003e\u003cp\u003eI am getting below run time error in Codeforces, though code works as expected in my local compiler. Time: 0 ms, memory: 0 KB Verdict: RUNTIME_ERROR I submitted my code for \u003ca href\u003d\"//codeforces.com/contest/962/problem/D\"\u003ehttp://codeforces.com/contest/962/problem/D\u003c/a\u003e\u003c/p\u003e\u003cp\u003eCode is below.\u003c/p\u003e\u003ch1\u003einclude\u0026lt;bits/stdc++.h\u0026gt;\u003c/h1\u003e\u003cp\u003eusing namespace std;\u003c/p\u003e\u003cp\u003eint main() {\u003c/p\u003e \u003cpre\u003e\u003ccode\u003eint n;\ncin \u0026gt;\u0026gt; n;\nvector\u0026lt;int\u0026gt;a(n);\nmap\u0026lt;int,set\u0026lt;int\u0026gt;\u0026gt;m;\nfor(int i\u003d0;i\u0026lt;n;i++)\n{\n    cin \u0026gt;\u0026gt; a[i];\n    m[a[i]].insert(i);\n}\nmap\u0026lt;int,set\u0026lt;int\u0026gt;\u0026gt;::iterator it;\nint flag\u003d1;\nwhile(flag)\n{\n    flag\u003d0;\n    for(it\u003dm.begin();it!\u003dm.end();it++)\n       if(it-\u0026gt;second.size()\u0026gt;\u003d2)\n       {\n         flag\u003d1;\n         break;\n       }\n    if(flag\u003d\u003d0)\n       break;\n    set\u0026lt;int\u0026gt;:: iterator it1;\n    set\u0026lt;int\u0026gt;s1;\n    s1\u003dit-\u0026gt;second;\n    it1 \u003d s1.begin();\n    int val \u003d it-\u0026gt;first;\n    int gh \u003d *it1;\n    s1.erase(it1);\n    it1++;\n    int kl \u003d *it1;\n    s1.erase(it1);\n    if(s1.size()\u003d\u003d0)\n       m.erase(val);\n    else\n       it-\u0026gt;second\u003ds1;\n    val*\u003d2;\n    m[val].insert(kl);\n}\nvector\u0026lt;int\u0026gt;ans(n,-1);\nint cnt\u003d0;\nfor(it\u003dm.begin();it!\u003dm.end();it++)\n{\n    set\u0026lt;int\u0026gt;::iterator it1;\n    set\u0026lt;int\u0026gt;s2;\n    s2\u003dit-\u0026gt;second;\n    it1\u003ds2.begin();\n    int val \u003d it-\u0026gt;first;\n    for(it1\u003ds2.begin();it1!\u003ds2.end();it1++)\n    {\n       ans[*it1]\u003dval;\n       cnt++;\n    }\n}\ncout \u0026lt;\u0026lt; cnt \u0026lt;\u0026lt; \u0026quot;\\n\u0026quot;;\nfor(int i\u003d0;i\u0026lt;n;i++)\n    if(ans[i]!\u003d-1)\n    cout \u0026lt;\u0026lt; ans[i] \u0026lt;\u0026lt; \u0026quot; \u0026quot;;\nreturn 0;\n\u003c/code\u003e\u003c/pre\u003e\u003cp\u003e}\u003c/p\u003e\u003cp\u003eUnable to understand the error. Please suggest how to solve it.\u003c/p\u003e\u003c/div\u003e","tags":["mysterious runtime error","c++14"]}}